media-source-respec.html

```diff
@@ -705,7 +705,8 @@
           <h4>Seeking</h4>
           <p>Run the following steps as part of the "<i>Wait until the user agent has established whether or not the media data for the new playback position is available, and, if it is, until it has decoded enough data to play back that position"</i> step of the <a def-id="hme-seek-algorithm"></a>:</p>
           <ol>
-            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>.
+            <li>The media element looks for <a def-id="media-segments"></a> containing the <var>new playback position</var> in each <a>SourceBuffer</a> object in <a def-id="activeSourceBuffers"></a>. Any position within a <a def-id="timerange"></a> in the current value of the <a def-id="hme-buffered"></a> attribute MUST have a buffered media segment for that position.
```

Comment:
I reworded some of the existing steps to make them more step-like and retain this note about 'ended' as a non-normative note since the steps are now normatively clear about this. See https://github.com/w3c/media-source/pull/151/commits/1e0c2763edd0841dc06ce474ca53aeef0f04ff23, below.